ArchiSteamFarm/Steam/Integration/SteamChatMessage.cs

```diff
@@ -152,6 +166,10 @@ internal static class SteamChatMessage {
 						continue;
 					}
 
+					if (stringReader.Peek() != -1 && messagePart[messagePart.Length-1] != ContinuationCharacter) {
```

Comment:
I'm not sure if that check for `messagePart[messagePart.Length-1] != ContinuationCharacter` is needed, the logic above already ensures us that we've dealt with splitting on continuation char, therefore we have (raw message):

- `aa`, no escaping.
- `\[` escaped some character.
- `\\` escaped backslash (same as above, but different char).

In all 3 cases we can include the paragraph char, as we don't have a "trailing escape char", correct me if I'm wrong.

Reply:
Okay, now I understood what it's for, it's for detecting if we're splitting mid-line.

This seems alright then, you can use `messagePart[^1]` instead of `messagePart[messagePart.Length-1]` for clarity.